.github/scripts/transform-message.ts

```diff
@@ -18,244 +19,163 @@ interface GitHubAction
+const eventHandlers: Record<string, EventHandler> = {
+  push: ({
+    event: { head_commit },
+    ref,
+    ref_name,
+    server_url,
+    repository,
+    actor,
+  }) => ({
+    title: "GitHub 代码提交",
+    content: [
+      [
+        createText("提交链接："),
+        createLink(head_commit!.url),
+      ],
+      [
+        createText("代码分支："),
+        createLink(ref || "", `${server_url}/${repository}/tree/${ref_name}`),
+      ],
+      [createText("提交作者："), createLink(actor, `${server_url}/${actor}`)],
+      [createText("提交信息："), createText(head_commit?.message || "")],
+    ],
+  }),
+
+  issues: ({ event: { issue } }, actionText) => ({
+    title: `GitHub issue ${actionText}：${issue?.title}`,
+    content: [
+      [createText("链接："), createLink(issue?.html_url, issue?.html_url)],
+      [
+        createText("作者："),
+        createLink(issue?.user?.login, issue?.user?.html_url),
+      ],
+      [
+        createText("指派："),
+        issue?.assignee
+          ? createLink(issue.assignee.login, issue.assignee.html_url)
+          : createText("无"),
+      ],
+      [
+        createText("标签："),
+        createText(issue?.labels?.map(({ name }) => name).join(", ") || "无"),
+      ],
+      [createText("里程碑："), createText(issue?.milestone?.title || "无")],
+      [createText("描述："), createText(issue?.body || "无")],
+    ],
+  }),
+
+  pull_request: ({ event: { pull_request } }, actionText) => ({
+    title: `GitHub PR ${actionText}：${pull_request?.title}`,
+    content: [
+      [
+        createText("链接："),
+        createLink(pull_request?.html_url, pull_request?.html_url),
+      ],
+      [
+        createText("作者："),
+        createLink(pull_request?.user.login, pull_request?.user.html_url),
+      ],
+      [
+        createText("指派："),
+        pull_request?.assignee
+          ? createLink(
+              pull_request.assignee.login,
+              pull_request.assignee.html_url
+            )
+          : createText("无"),
+      ],
+      [
+        createText("标签："),
+        createText(
+          pull_request?.labels?.map(({ name }) => name).join(", ") || "无"
+        ),
+      ],
+      [
+        createText("里程碑："),
+        createText(pull_request?.milestone?.title || "无"),
+      ],
+      [createText("描述："), createText(pull_request?.body || "无描述")],
+    ],
+  }),
+
+  comment: ({ event: { comment, issue, discussion } }, actionText) => {
+    const title = issue?.title || discussion?.title || "未知帖子";
+    return {
+      title: `GitHub 帖子评论：${title}`,
+      content: [
+        createContentItem("链接：", comment?.html_url) as [any, any],
+        createContentItem("作者：", comment?.user ? createUserLink(comment.user) : createText("无")) as [any, any],
+        createContentItem("描述：", comment?.body || "无描述") as [any, any]
+      ],
+    };
+  },
+
+  release: ({ event: { release } }, actionText) => {
+    const title = release?.name || release?.tag_name || "未知版本";
+    return {
+      title: `GitHub Release 发布：${title}`,
```

Comment:
```suggestion
  release: ({ event: { release } }, actionText) => ({
      title: `GitHub Release 发布：${release!.name || release!.tag_name}`,
```
在这种事件下，它的信息对象是一定存在的。